doc/adr/0009-api-key.md

```diff
@@ -0,0 +1,26 @@
+# 9. API Keys
+
+Date: 2022-07-06
+
+## Status
+
+Accepted
+
+## Context
+
+We want to filter incoming requests in the UseID backend that only client with an approved integration could access UseID endpoints.
+
+## Decision
+
+We use Bearer authentication. The Client (eService) should add a token in the Authorization header when requesting to the endpoints and the token should be a random string of 51 characters.
+
+e.g.
+
+`Authorization: Bearer <token>`
+
+The token should, for MVP purpose, be configured as environment variable. For the future releases, a lifecycle of token should be defined, e.g. validity of the token.
```

Comment:
I would remove the whole sentence since it is an implementation detail which is not relevant for the decision.